suse_migration_services/units/kernel_load.py

```diff
@@ -36,16 +37,23 @@ def main():
     """
     root_path = Defaults.get_system_root_path()
 
-    target_kernel = Defaults.get_target_kernel()
-    target_initrd = Defaults.get_target_initrd()
+    target_kernel = os.sep.join([root_path, Defaults.get_target_kernel()])
+    target_initrd = os.sep.join([root_path, Defaults.get_target_initrd()])
+    in_memory_path = os.sep
```

Comment:
I think this is hard to understand without knowing the context of the system underneath. The code doesn't know that it runs in a kiwi live system loaded into RAM. So I would name that differently and I think we should stick it to ```/var/tmp/kexec/``` , maybe something like:

```python
kexec_boot_data = '/var/tmp/kexec'
Path.create(kexec_boot)

....
```